### ISSUE TITLE
Missing size parameter in object target file

### ISSUE THREAD

**Issue by hobinjk-ptc** [ptcrealitylab/vuforia-spatial-edge-server#114]
```
TypeError: Cannot read property 'split' of undefined
    at /Users/James/re/vuforia-spatial-edge-server/libraries/utilities.js:294:61
    at Parser.<anonymous> (/Users/James/re/vuforia-spatial-edge-server/node_modules/xml2js/lib/parser.js:304:18)
    at emitOne (events.js:116:13)
    at Parser.emit (events.js:211:7)
    at SAXParser.onclosetag (/Users/James/re/vuforia-spatial-edge-server/node_modules/xml2js/lib/parser.js:262:26)
    at emit (/Users/James/re/vuforia-spatial-edge-server/node_modules/sax/lib/sax.js:624:35)
    at emitNode (/Users/James/re/vuforia-spatial-edge-server/node_modules/sax/lib/sax.js:629:5)
    at closeTag (/Users/James/re/vuforia-spatial-edge-server/node_modules/sax/lib/sax.js:889:7)
    at SAXParser.write (/Users/James/re/vuforia-spatial-edge-server/node_modules/sax/lib/sax.js:1436:13)
    at Parser.exports.Parser.Parser.parseString (/Users/James/re/vuforia-spatial-edge-server/node_modules/xml2js/lib/parser.js:323:31)
```

@benptc Any insight?

**Comment by hobinjk-ptc** [comment]
Specifically the .xml is just ```
<?xml version="1.0" encoding="UTF-8"?>
<QCARConfig xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="QCAR_Dataset_config.xsd">
  <Tracking>
    <ObjectTarget name="yarn3Rb543is2uut" />
  </Tracking>
</QCARConfig>
```

**Comment by benptc** [comment]
I ran into that same bug recently and I'm not sure how we didn't run into it before. I think we should just catch the error and return the default size if so.

**Comment by benptc** [comment]
It's possible we only ever tested the object tracker with the feeder machine objects, whose XML files may have been manually edited to include size when we first incorporated this.

**Comment by benptc** [comment]
Fixed with PR #116
